packages/styles/scss/css-variables/variables.scss

```diff
@@ -116,8 +116,11 @@
   @include light-dark('support-caution-major', rgb-color('orange', '500'), rgb-color('orange', '400'));
   @include light-dark('support-caution-undefined', rgb-color('purple', '500'), rgb-color('purple', '400'));
 
-  // Spacing
+  // Spacing  
   @include spacing.emit-spacing-custom-properties();
+
+  // Hover colors
+  @include light-dark('hover-red', rgb-color('red', '500'), rgb-color('red', '600'));
```

Comment:
`hover-red` should probably be named as a component token instead of this generic name. Since you have `metrics-red` as a variable already I think it makes more sense to name it `metric-red-hover`.